src/test/javascript/spec/component/lecture-unit/text-unit/text-unit-form.component.spec.ts

```diff
@@ -0,0 +1,182 @@
+import * as chai from 'chai';
+import * as sinonChai from 'sinon-chai';
+import * as sinon from 'sinon';
+import { SinonStub } from 'sinon';
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { TextUnitFormComponent, TextUnitFormData } from 'app/lecture/lecture-unit/lecture-unit-management/text-unit-form/text-unit-form.component';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { TranslatePipe, TranslateService } from '@ngx-translate/core';
+import { MockComponent, MockPipe, MockProvider } from 'ng-mocks';
+import { FormDateTimePickerComponent } from 'app/shared/date-time-picker/date-time-picker.component';
+import * as moment from 'moment';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { MockRouter } from '../../../helpers/mocks/mock-router';
+import { Router } from '@angular/router';
+
+chai.use(sinonChai);
+const expect = chai.expect;
+
+@Component({ selector: 'jhi-markdown-editor', template: '' })
+class MarkdownEditorStubComponent {
+    @Input() markdown: string;
+    @Input() enableResize = false;
+    @Output() markdownChange = new EventEmitter<string>();
+}
+
+describe('TextUnitFormComponent', () => {
+    let store = {};
+    const sandbox = sinon.createSandbox();
+    let textUnitFormComponentFixture: ComponentFixture<TextUnitFormComponent>;
+    let textUnitFormComponent: TextUnitFormComponent;
+    let getItemStub: SinonStub;
+    let removeItemStub: SinonStub;
+    let setItemStub: SinonStub;
+    beforeEach(() => {
+        // mocking router
+        // mocking the local storage for cache testing
+        store = {};
+        getItemStub = sandbox.stub(localStorage, 'getItem').callsFake((key: string) => {
+            return store[key] || null;
+        });
+        removeItemStub = sandbox.stub(localStorage, 'removeItem').callsFake((key: string) => {
```

Comment:
GitHub Actions: `'removeItemStub' is assigned a value but never used`

Reply:
d67c60cb6977715928970ad04df3a2f288f31f6b